test/MinterGateway.t.sol

```diff
@@ -2037,6 +2039,8 @@ contract MinterGatewayTests is TestUtils {
 
         uint240 activeOwedM_ = _minterGateway.activeOwedMOf(_minter1);
 
+        if (activeOwedM_ == 0) return; // No penalties to impose if no M is owed.
```

Comment:
`vm.assume(activeOwedM_ != 0); // No penalties to impose if no M is owed.`

Reply:
added